test/functionalTest/cases/0787_cprs_full_functional/update_forward_depends_on_attr_field_in_ngsiv2.test

```diff
@@ -581,11 +581,16 @@ Date: REGEX(.*)
 
 09. Update/UPDATE E1/A1+A2+A3+A4+A5+A6+A7 in CB
 ===============================================
-HTTP/1.1 204 No Content
-Content-Length: 0
+HTTP/1.1 404 Not Found
+Content-Length: 92
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
+    "description": "Attributes that were not updated: [ E1-T1 : [A7] ]",
```

Comment:
Looks better. I thought that `{` was a forbidden char, but I have re-checked and I was wrong.